app/controllers/hub/outbound_calls_controller.rb

```diff
@@ -19,6 +19,12 @@ def show
       @outbound_call = OutboundCall.find(params[:id])
     end
 
+    def update
+      @outbound_call = OutboundCall.find(params[:id])
+      @outbound_call.update(params.require(:outbound_call).permit(:note))
```

Comment:
curious about a couple of things:
- why are the permitted params here not separated into a separate method like the ones in `create`?
- why is the existing `permitted_params` method not private?

also, did github change something about where you can comment? i wanted to leave a comment on `create` above just to say "i don't understand it but i assume it has already been through review?" but it wouldn't let me :/

Reply:
I should leave a note on this method -- in create, we're creating the object and dialing the number to start the call. That then directs you to the show page that includes some info about the call and a "notes" textarea. If the user enters notes and then submits, the object will be updated with the note.